packages/twenty-utils/congratulate-dangerfile.ts

```diff
@@ -0,0 +1,72 @@
+import { danger } from 'danger';
+
+const ordinalSuffix = (number) => {
+  const v = number % 100;
+  if (v === 11 || v === 12 || v === 13) {
+    return number + 'th';
+  }
+  const suffixes = { 1: 'st', 2: 'nd', 3: 'rd' };
+  return number + (suffixes[v % 10] || 'th');
+};
+
+const fetchContributorStats = async (username: string) => {
+  const apiUrl = `https://twenty.com/api/contributors/contributorStats/${username}`;
+
+  const response = await fetch(apiUrl);
+  const data = await response.json();
+  return data;
+};
+
+const fetchContributorImage = async (username: string) => {
+  const apiUrl = `https://twenty.com/api/contributors/${username}/og.png`;
+
+  await fetch(apiUrl);
+};
+
+const runCongratulate = async () => {
+  const pullRequest = danger.github.pr;
+  const userName = pullRequest.user.login;
+
+  const { data: pullRequests } =
```

Comment:
Is this going to be posted on every PR? We want to discard the `assignableUsers` + a list (same as get-contributor activity / page.tsx - search 'nimraahmed' ; there should be just one source of truth for non-contributors)